controllers/kyma_controller_test.go

```diff
@@ -67,7 +67,7 @@ var _ = Describe("Kyma with empty ModuleTemplate", Ordered, func() {
 		Expect(
 			kymaInCluster.ContainsCondition(v1beta2.ConditionTypeModuleCatalog, metav1.ConditionTrue)).To(BeTrue())
 		By("Module Catalog created")
-		Eventually(ModuleTemplatesExist(controlPlaneClient, kyma),
+		Eventually(ModuleTemplatesExist(controlPlaneClient, kyma, kyma.GetNamespace()),
```

Comment:
This should not be kyma namespace, if this test pass, then should be a coincidence.